runtime/doc/deprecated.txt

```diff
@@ -118,6 +118,10 @@ LSP Functions ~
 *vim.lsp.util.set_loclist()*		Use |setloclist()| instead.
 *vim.lsp.buf_get_clients()*		Use |vim.lsp.get_active_clients()| with
                                         {buffer = bufnr} instead.
+*vim.lsp.buf.formatting()*		Use |vim.lsp.buf.format()| with
+					{async = true} instead.
+*vim.lsp.buf.range_formatting()*	Use |vim.lsp.formatexpr()|
```

Comment:
there also `vim.lsp.buf.format` not `vim.lsp.formatexpr`. it is set to `formatexpr` option

Reply:
I typo'd the second line of the range_formatting replacements, which I'll fix. It should read `vim.lsp.buf.format()` instead of `vim.lsp.format()`.

As for `vim.lsp.formatexpr`, it does seem to be a valid function and is recommended by the deprecation warning when calling `vim.lsp.buf.range_formatting()`.

Reply:
I'm sorry, I'm not sure I understand what you're saying. Can you clarify? Thanks

Reply:
Oh damn I was too quick with the merge, I missed the `vim.lsp.format` / `vim.lsp.buf.format`. Can you create a PR with a fix for that?

Reply:
Looks like it was resolved by another PR